From 58378753abc49a8965d679b82e7fecc4349a06f8 Mon Sep 17 00:00:00 2001
From: Justin Schneck <jschneck@mac.com>
Date: Thu, 25 Jun 2020 13:36:35 -0400
Subject: [PATCH] force older version of libgbm

---
 libweston/backend-drm/meson.build | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/libweston/backend-drm/meson.build b/libweston/backend-drm/meson.build
index a7f6296..29af19a 100644
--- a/libweston/backend-drm/meson.build
+++ b/libweston/backend-drm/meson.build
@@ -46,12 +46,6 @@ if get_option('renderer-gl')
 	if not dep_gbm.found()
 		error('drm-backend with GL renderer requires gbm which was not found. Or, you can use \'-Drenderer-gl=false\'.')
 	endif
-	if dep_gbm.version().version_compare('>= 17.1')
-		config_h.set('HAVE_GBM_MODIFIERS', '1')
-	endif
-	if dep_gbm.version().version_compare('>= 17.2')
-		config_h.set('HAVE_GBM_FD_IMPORT', '1')
-	endif
 	deps_drm += dep_gbm
 	srcs_drm += 'drm-gbm.c'
 	config_h.set('BUILD_DRM_GBM', '1')
-- 
2.25.1

